chapters/fr/chapter1/2.mdx

```diff
@@ -0,0 +1,21 @@
+# Traitement du language naturel (NLP)
+
+Avant de commencer avec les modèles Transformer, nous allons voir succinctement ce qu'est le traitement du langage naturel et pourquoi il est important.
+
+## NLP, qu'est ce que c'est?
```

Comment:
Should we use `Traitement du langage naturel` and it's French acronym `TLN` or should I keep `NLP` because to be honest everyone use `NLP` and not `TLN` in French. It's always easier to keep the english version to talk to people about a subject. But the question needs to be asked for this course translation :)

Reply:
Hehe, we've had the same discussion for the other languages :) I think we can follow the general rule of adopting the most common usage - e.g. in some languages it might be NLP, in others the direct translation. 

So for French, I like your idea to translate `Natural language processing` and keep `NLP` fixed.